scripts/cli-docs-generate.sh

```diff
@@ -13,15 +13,24 @@ go run internal/cli/docs/generate-cli-docs.go "${destination}"/
 # Fix ${HOME} references to proper `~`.
 perl -pi -e "s@${HOME}@~@" "${destination}"/*md
 
-# Fix the cross-links to match hierarchy and mdbook expectations.
-sed -i 's@(\.\./\(.*\))@(\1.md)@' "${destination}"/*md
+# Fix the cross-links to match hierarchy and tooling expectations.
+sed -i 's@(\.\./\(.*\))@(./\1.md)@' "${destination}"/*md
 
-# Drop the HUGO specific annotations from file heads
+# Rework the YAML header/annotations. Fix `push` link.
 for md in "${destination}"/*md
 do
-    tail --lines +6 $md > $$
+    echo -n .
+    ( cat $md | grep -v linkTitle: | grep -v weight:
+    ) | sed -e 's|epinio_app_push.md|epinio_push.md|' \
+      > $$
     mv $$ $md
 done
 
+# Fix up the app chart docs, hide the operator commands
+#rm "${destination}"/epinio_app_chart_create.md
+#rm "${destination}"/epinio_app_chart_delete.md
+#cat "${destination}"/epinio_app_chart.md | grep -v _create | grep -v _delete > $$
+#mv $$ "${destination}"/epinio_app_chart.md
+
```

Comment:
```suggestion
```

Reply:
Accepted and committed. @jimmykarily, this has dismissed your approval.